packages/marketplace/src/styles/blocks/diff-render-html.scss

```diff
@@ -42,6 +44,7 @@
 }
 
 .block {
+  word-wrap: break-word;
```

Comment:
I think `word-wrap` is now changed to `overflow-wrap` 
https://stackoverflow.com/a/15137272/7811940